Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1907,6 +1907,37 @@ def refresh_session_command():
     return create_entry_object(contents=response, hr=f'CrowdStrike Session Refreshed: {session_id}')
 
 
+def build_url_filter_for_device_id(args):
+    indicator_type = args.get('type')
+    indicator_value = args.get('value')
+    url_filter = f'/indicators/queries/devices/v1?type={indicator_type}&value={indicator_value}'
+    return url_filter
+
+
+def handle_errors(raw_res):
```

Comment:
Let's make this a function that gets an error message, say "format_error_message", and then return_error the error message wherever it is used.
Otherwise it's a bit confusing